src/field_5x52_impl.h

```diff
@@ -263,7 +263,8 @@ SECP256K1_INLINE static int secp256k1_fe_is_odd(const secp256k1_fe *a) {
 SECP256K1_INLINE static void secp256k1_fe_clear(secp256k1_fe *a) {
     int i;
 #ifdef VERIFY
-    secp256k1_fe_verify_init(a);
+    a->magnitude = 0;
```

Comment:
It isn't actually a straight inline. Both values are set to zero as part of the cleanse. The thought was to be a bit pedantic about showing the progression in the commits, but it might have been be overboard.

In the new commit progression with `secp256_fe_set_zero()`, seeing it side-by-side with `secp256k1_fe_clear()`, with `a->normalized = 1;` and `a->normalized = 0;` respectively is clearer with the intent of the two separated functions.